lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -496,6 +502,13 @@ export abstract class ClientApplication {
         this.eventHandler.removeEventCallback(callbackId);
     }
 
+    /**
+     * Gets the token cache for the application.
+     */
+    getTokenCache(): TokenCache {
```

Comment:
We should add this to `IPublicClientApplication` as well.

Reply:
And I think the return type should `ITokenCache`

Reply:
Done.